install_config/install/prerequisites.adoc

```diff
@@ -121,6 +121,14 @@ Therefore, the recommended size of a cluster of 2000 pods would be 20 CPU cores
 and 50 GB of RAM, as well as the minimum requirements for a master host of 2 CPU
 cores and 8 GB of RAM.
 
+[NOTE]
+====
+It is important to understand that oversubscribing the physical resources on a
```

Comment:
The wording is awkward to my ear (although i can see that the text is similar to that above, which does not sound awkward, strangely enough).  Suggest changing the admonition to `IMPORTANT`, and chopping the blurb head to start w/ "Oversubscribing the physical resources...".